janet.json

```diff
@@ -5,6 +5,6 @@
     "*"
   ],
   "exposure": [
-    "wix.com"
+    "bo"
```

Comment:
Why not wix.com?

Reply:
It didn't work at the beginning, so I tried to change to bo.
But this also doesn't work 🤷‍♂️